api/envoy/config/route/v3/route.proto

```diff
@@ -169,4 +169,7 @@ message Vhds {
 
   // Configuration source specifier for VHDS.
   core.v3.ConfigSource config_source = 1 [(validate.rules).message = {required: true}];
+
+  // Pass the route name in the initial subscription request instead of a wildcard subscription.
+  bool use_singleton_subscription = 2;
```

Comment:
cc @markdroth 
The current implementation of VHDS sends a wildcard subscription, and if on-demand is used, then additional single-resource subscriptions will be used.
This flag is suggested to prevent sending the wildcard in the first place. WDYT?